docs/guides/data/storing_output.md

```diff
@@ -1,29 +1,85 @@
+and can be accessed and reused later.
 
-#### Example 1
+The most common thing users save is model checkpoints, but anything that ends
+up in the `/output` directory at the end of a run will be saved (use your
+imagination!).
+
+Let's work through a couple of examples to see how to save data during a run.
 
-This job runs a Python script called `helloworld.py` and redirects its output logs into the file `/output/my-output-file.txt`:
+#### Example 1
+This job prints the string "Hello, world!", and saves it to a file called
+`hello.txt`. Because `hello.txt` is located in the `/output` directory, it will
+be saved and available after the job finishes:
 
 ```bash
-$ floyd init quick-start
-$ floyd run "python helloworld.py > /output/my-output-file.txt"
+$ floyd init my_awesome_hello_world_project
+$ floyd run "echo 'Hello, world!' > /output/hello.txt"
 Syncing code ...
-...
 ```
 
-*NOTE*: This is a simple example that only stores the standard output of `helloworld.py`. Since the file is created under the special `/output` directory, it will be saved even after your job has ended. 
+!!! note
+    If you are not familiar with what
+    `echo 'Hello, world!' > /output/hello.txt` does, here's a quick
+    explanation:
 
+    - The `echo 'Hello, world!'` part outputs the string `Hello, world!`.
+    - The `>` part of the command redirects the printed output of `echo 'Hello, world!'`
+      (which is, of course, `Hello world!`) to the file specified after the `>`.
+    - The `/output/hello.txt` part of the command specifies where the `Hello,
+      world!` should be written to: `/output/hello.txt`. Because `hello.txt` is
+      in the `/output` directory, it will be preserved for future reference and
+      use.
 
 #### Example 2
+In this example, we'll use Python to save some data to a file in the `/output`
+directory. Put this code in a file named `save_example.py`:
 
-If you want to save and retain your model checkpoints and other data inside your code, you should write them to `/output`.
+```python
+with open('/output/myfile.txt', 'a') as f:
+    f.write('Please save me!\n')
+```
+
+If you run this code locally on your computer, you'll probably get something
+like this:
+
+```bash
+Traceback (most recent call last):
+  File "save_example.py", line 1, in <module>
+    with open('/output/myfile.txt', 'a') as f:
+IOError: [Errno 2] No such file or directory: '/output/myfile.txt'
+```
 
-Here is a sample Tensorflow example that saves the model weights:
+That's because there is no `/output` directory on your computer. In contrast,
+every job on FloydHub runs on a server that has a `/output` directory, so the
+command won't fail on FloydHub. Let's run it with the following commands:
 
-`save_restore_model.py`
 ```bash
+$ floyd init save_example_2
+$ floyd run "python save_example.py"
+Creating project run. Total upload size: 267.0B
+Syncing code ...
+```
+Success! We can now view the output, download it, or even use it again in
+future jobs.
+
```

Comment:
Can you please add a section on How to view output data (both dashboard and CLI) and link that in the examples here